apps/admin/src/containers/Purchase/index.tsx

```diff
@@ -36,8 +36,13 @@ export const Purchase = () => {
     setIsProcessing(true);
     setForceValidate(true);
     throw new Error("Just because it compiles, doesn't mean it works.  Check properly");
```

Comment:
Note the throw.  This code is just filler - not meant to be used without actual effort to fix